packages/block-editor/src/hooks/block-hooks.scss

```diff
@@ -13,4 +13,10 @@
 	.components-toggle-control .components-h-stack .components-h-stack {
 		flex-direction: row;
 	}
+
+	.block-editor-hooks__block-hooks-helptext {
+		color: $gray-700;
+		font-size: $helptext-font-size;
+		margin-bottom: $grid-unit-20;
+	}
```

Comment:
I stole this from https://github.com/WordPress/gutenberg/blob/7808dc722f3bd5e9df4775fcd4aebeefbbaf2a9c/packages/block-editor/src/hooks/layout.scss#L25-L29.